docs/getting-started/getting-started.mdx

```diff
@@ -5,12 +5,9 @@ description: "Preparing your machine"
 
 ## Prerequisites
 
-There are a few packages that need to be installed in order to run 01OS on your computer
+To run 01OS on your computer, you will need to install a few essential packages.
 
-```bash
-# Install poetry
-curl -sSL https://install.python-poetry.org | python3 -
-```
+Install [Poetry](https://python-poetry.org/).
```

Comment:
Seeing it in mintlify I think "Install poetry" might be missed, maybe this should be more prominent like a section? Like "Install Poetry", then "To install poetry, follow the official guide here...". I think linking to the installation guide too would be faster for folks: https://python-poetry.org/docs/#installing-with-the-official-installer

<img width="300" alt="Screenshot 2024-07-29 at 8 34 32 PM" src="https://github.com/user-attachments/assets/d7429a99-bf86-4422-973f-ed1b98e97a4c">

Reply:
Could try a big [card](https://mintlify.com/docs/content/components/cards) to draw attention to it, if you think it looks right (i love cards)